ReactAndroid/src/main/java/com/facebook/react/modules/deviceinfo/DeviceInfoModule.java

```diff
@@ -85,8 +90,7 @@ public void emitUpdateDimensionsEvent() {
 
     if (mReactApplicationContext.hasActiveReactInstance()) {
       // Don't emit an event to JS if the dimensions haven't changed
-      WritableNativeMap displayMetrics =
-          DisplayMetricsHolder.getDisplayMetricsNativeMap(mFontScale);
+      WritableMap displayMetrics = DisplayMetricsHolder.getDisplayMetricsWritableMap(mFontScale);
```

Comment:
What was the logic behind `WritableNativeMap` and `WritableMap`? Sorry I'm not familiar with the difference

Reply:
`WritableNativeMap` is an implementation of `WritableMap` (and that is the implementation that is returned from `DisplayMetricsHolder`). But in Java-only test code, `WritableNativeMap` is not supported and needs to be switched out for `JavaOnlyMap`. So in order to test this class, it needs to depend on `WritableMap` instead.